distributor-node/src/services/networking/NetworkingService.ts

```diff
@@ -437,16 +447,16 @@ export class NetworkingService {
   }
 
   async getQueryNodeStatus(): Promise<StatusResponse['queryNodeStatus']> {
-    const qnState = await this.queryNodeApi.getQueryNodeState()
+    const squidStatus = await this.queryNodeApi.getQueryNodeState()
 
-    if (qnState === null) {
-      this.logger.error("Couldn't fetch the state from connected query-node")
+    if (squidStatus === null) {
+      this.logger.error("Couldn't fetch the state from connected storage-squid")
     }
 
     return {
       url: this.config.endpoints.queryNode,
-      chainHead: qnState?.chainHead || 0,
-      blocksProcessed: qnState?.lastCompleteBlock || 0,
+      chainHead: (await this.runtimeApi.derive.chain.bestNumber()).toNumber() || 0,
```

Comment:
Possibly worth try-catching as well so if `runtimeApi` is unavailable we get a clear error message?